src/main/webapp/assets/script.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google LLC
```

Comment:
So I know that the linter is broken on master until my PR is merged, but could I show you how to add local linting in VS Code using the same rule sheet that the github action linter uses? That way it'll be easy to catch small stuff like using 4 spaces to indent instead of 2, formatting JSDoc comments, stuff like that!

Reply:
Yeah, that'd be great! I'll dm you!